src/WebApp/Startup.cs

```diff
@@ -97,6 +123,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
 
             app.UseRouting();
 
+            var localizationOptions = app.ApplicationServices.GetService<IOptions<RequestLocalizationOptions>>().Value;
+            app.UseRequestLocalization(localizationOptions);
```

Comment:
I think you can just use the following, it should pick up the options automatically. (I can't test this right now.)

~~~csharp
app.UseRequestLocalization();
~~~

Reply:
I've tested this, it is able to pick up the options from the services collection.
```suggestion
            app.UseRequestLocalization();
```